data/iconbrowser.appdata.xml.in

```diff
@@ -59,10 +59,10 @@
     <release version="1.2.0" date="2020-07-29">
       <description>
         <ul>
-          <li>Add audio-headphones and audio-headset</li>
-          <li>Add distribute actions</li>
+          <li>Add "audio-headphones" and "audio-headset"</li>
+          <li>Add GitHub Actions for distribution</li>
           <li>Disable sidebar when searching</li>
-          <li>Remove non-fdo app names</li>
+          <li>Remove freedesktop.org non-compliant app icons</li>
```

Comment:
Maybe it would be more natural sounding to say something like

```suggestion
          <li>Remove app icons that aren't in the freedesktop.org specification</li>
```

Reply:
Nice, it would be even clearer. Thank you!